compare_experiments/compare_generic_genes.py

```diff
@@ -0,0 +1,324 @@
+
+f1 = sns.distplot(template_1.mean()[concordant_genes], kde=False)
+f1.set_title(f"Expression of concordant genes in {project_id1}")
+f1.set_xlabel("log(gene expression)")
+f1.set_ylabel("log(count)")
+f1.set(xscale="log", yscale="log")
+
+# +
+# Distribution of concordant genes in template experiment 2
+template2_mean = template_2.mean()
+print(
+    "Percent concordant genes with 0 expression in template 2:",
+    len(template2_mean[concordant_genes].loc[template2_mean[concordant_genes] == 0])
+    / len(template2_mean[concordant_genes]),
+)
+
+print(
+    "Percent nonzero concordant genes in template 2:",
+    len(
+        template2_mean[concordant_genes].loc[
+            (template2_mean[concordant_genes] > 0)
+            & (template2_mean[concordant_genes] < 1000)
+        ]
+    )
+    / len(template2_mean[concordant_genes]),
+)
+
+# There are more 0 expressed genes in this template experiment
+f2 = sns.distplot(template_2.mean()[concordant_genes], kde=False)
+f2.set_title(f"Expression of concordant genes in {project_id2}")
+f2.set_xlabel("log(gene expression)")
+f2.set_ylabel("log(count)")
+f2.set(xscale="log", yscale="log")
+
+# +
+# Distribution of discordant gense in template experiment 1
+template1_mean = template_1.mean()
+
+print(
+    "Percent discordant genes with 0 expression in template 1:",
+    len(template1_mean[discordant_genes].loc[template1_mean[discordant_genes] == 0])
+    / len(template1_mean[discordant_genes]),
+)
+
+print(
+    "Percent nonzero discordant genes in template 1:",
+    len(
+        template1_mean[discordant_genes].loc[
+            (template1_mean[discordant_genes] > 0)
+            & (template1_mean[discordant_genes] < 1000)
+        ]
+    )
+    / len(template1_mean[discordant_genes]),
+)
+
+print(
+    len(template1_mean[discordant_genes].loc[template1_mean[discordant_genes] > 0])
+    / len(template1_mean[discordant_genes])
+)
+f3 = sns.distplot(template_1.mean()[discordant_genes], kde=False)
+f3.set_title(f"Expression of discordant genes in {project_id1}")
+f3.set_xlabel("log(gene expression)")
+f3.set_ylabel("log(count)")
+f3.set(xscale="log", yscale="log")
+
+# +
+# Distribution of discordant genes in template experiment 2
+template2_mean = template_2.mean()
+
+print(
+    "Percent discordant genes with 0 expression in template 2:",
+    len(template2_mean[discordant_genes].loc[template2_mean[discordant_genes] == 0])
+    / len(template2_mean[discordant_genes]),
+)
+
+print(
+    "Percent nonzero discordant genes in template 2:",
+    len(
+        template2_mean[discordant_genes].loc[
+            (template2_mean[discordant_genes] > 0)
+            & (template2_mean[discordant_genes] < 1000)
+        ]
+    )
+    / len(template2_mean[discordant_genes]),
+)
+
+f4 = sns.distplot(template_2.mean()[discordant_genes], kde=False)
+f4.set_title(f"Expression of discordant genes in {project_id2}")
+f4.set_xlabel("log(gene expression)")
+f4.set_ylabel("log(count)")
+f4.set(xscale="log", yscale="log")
```

Comment:
Are the boxes different sizes because this is a log-log plot? Or is there just a consistant number of genes in the low-expression region?

Reply:
Looks like its because of the scale